lib/fs/casefs.go

```diff
@@ -29,7 +29,7 @@ type ErrCaseConflict struct {
 }
 
 func (e *ErrCaseConflict) Error() string {
-	return fmt.Sprintf(`given name "%v" differs from name in filesystem "%v"`, e.Given, e.Real)
+	return fmt.Sprintf(`remote "%v" uses different upper or lowercase characters than local "%v"; change the casing on either side to fix`, e.Given, e.Real)
```

Comment:
While pedantic, it currently doesn't say that you have to change the casing such that the two are equal. Should be obvious, but if it can be helped, nothing should be considered obvious :) 

```suggestion
	return fmt.Sprintf(`remote "%v" uses different upper or lowercase characters than local "%v"; change the casing on either side to match the other`, e.Given, e.Real)
```

Reply:
Fixed 👍.